SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -12,6 +12,147 @@ The content for this appendix may be pushed to SDPi 1.x.
+
+Though a full explanation of the this framework is beyond this discussion footnote:hanging_gardens_framework_article[A more detailed explanation of this model is provided on the https://confluence.hl7.org/x/4ijxB[IHE-HL7 Gemini _Hanging Gardens Framework_ confluence page].  Last accessed 2022.10.04.], some general observations will help understand the value and use of the framework:
+
+. Real-world Requirements are at the top -- use case requirements, detailing clinical system functions ensure that all requirements and capabilities are rooted in the purposes that technology users are advancing, namely to provide safe
+. <inter layer interfaces Requirements -> Capabilities API, Req Def -> Req Usage >
+. <Standards groups / families - internally coherent ... hopefully!>
+. <Vertical / horizontal>
+. <profiles navigate from the top to the bottom similar with each layer contributing to those below and converging in a single interface capability bundle>
+. <physical layers are leveraged / not re-invented!>
+. <requirements interoperability traces "profile lines" from top to bottom>
+. <requirements TYPEs are rooted in the layers that they represent and link Defs to Usage, see ##TODO:  ADD LINK HERE to "SDPi Requirements Core Model"" ## section below.>
+
+#TODO:  ADD INTEROPERABILITY PATHS EXAMPLE VIA A TABLE (VS. LINES THROUGH GRAPHIC)#
+
+Note:  The implementation of this framework will be extended as the specifications and tooling mature.
+
 // A.2
 === Integrating Safety, Effectiveness & Security Requirements & Considerations
 
+In 2007, a joint effort between ISO/TC 215 and IEC/SC 62A was launched --- Joint Working Group 7 (JWG7) -- to focus on how to apply risk management to medical devices and health information systems and software that needed to interoperate on shared (hospital owned & managed) networking infrastructure.  The resulting standard, <<iec_80001_1_2021>>, with a first edition published in 2010 and revised in 2021, not only provided a process for performing coordinated multi-stakeholder risk management for these technologies, but recognized the three *key properties* that would be the focus of that risk management:  *_Safety, Effectiveness & Security (SES)_* footnote:ses_definitions[For definitions of these and other related terms, consult the https://81001.org[NHS 81001.org web page.] Last accessed 2022.10.04.].
+
+During the development of the 80001-1 standard, though, it was recognized that risk management is just one of a number of other core themes that had to be managed in concert (e.g., quality management, human factors / usability).  Also ensuring SES required processes that involve a total product lifecycle, with responsibilities transitioning across that period.  To address these requirements, JWG7 developed the <<iso_81001_1_2021>> standard, which also included The Temple diagram to communicate the various aspects of SES that must be considered and managed:
+
+.ISO/IEC 81001-1 _The Temple Diagram_
+
```

Comment:
```suggestion
[#the_temple_diagram]
```

Reply:
See similar Reply question to Hanging Gardens Comment above.  Whatever is done there can be replicated here.